README.md

```diff
@@ -0,0 +1,58 @@
+#Deploying to Heroku
+
+First make sure you have Node.js (which comes with npm as well) installed. 
+Access to the heroku and gmail acc can be found in the Slack channel.
+
+Then install the Heroku CLI (command-line interface) which also comes with a Git installer. 
+
+The app dependencies have already been installed. It is under dependencies in the package.json file. 
+**The version of Node.js that we are currently using is 6.9.5 (subject to change).** It will be used to run the app on Heroku.

```

Comment:
That's the recommended one when you install Node. That being said, odd ones aren't stable yet, we can switch to 6.9.4 or any other stable one